### PR TITLE
Match include style in avify4mtest and avifenc

### PR DESCRIPTION
**Pull request by y-guyon**
Also describe ctest usage in CMakeLists.txt.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -539,7 +539,9 @@ endif()
 
 option(AVIF_BUILD_TESTS "Build avif tests." OFF)
 if(AVIF_BUILD_TESTS)
-    # Reference tests for CTest to work.
+    # With testing enabled, all targets referenced by add_test() can be run with
+    # CMake's ctest command line tool from the build folder: ctest [options]
+    # Options include filtering tests with a regexp -R, multithreading -j etc.
```
Comment (wantehchang) [inline CMakeLists.txt:544]: Thank you! I didn't realize that we can just say "ctest" without any options. Given that, we can omit everything after "from the build folder" if you prefer a concise comment.

File: CMakeLists.txt [hunk 2]
```diff
@@ -573,7 +575,8 @@ if(AVIF_BUILD_TESTS)
         set_target_properties(avify4mtest PROPERTIES LINKER_LANGUAGE "CXX")
     endif()
     target_link_libraries(avify4mtest avif ${AVIF_PLATFORM_LIBRARIES})
-    target_include_directories(avify4mtest PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+    target_include_directories(avify4mtest
+        PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/apps/shared)
```
Comment (wantehchang) [inline CMakeLists.txt:579]: Nit: `${CMAKE_CURRENT_SOURCE_DIR}/` can be omitted. Please compare with line 559. We should use the same approach at both places.

I am fine with either approach, but slightly prefer the more concise approach unless it is broken in some build configurations.

**Review by wantehchang — CHANGES_REQUESTED** [review]
LGTM. I have a suggested change.

File: CMakeLists.txt (hunk 2 above)
Comment (y-guyon) [reply CMakeLists.txt:579]: Indeed, from [CMake doc](https://cmake.org/cmake/help/latest/command/target_include_directories.html):

> A relative path will be interpreted as relative to the current source directory (i.e. CMAKE_CURRENT_SOURCE_DIR)

**Review by wantehchang — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**